types/slonik/index.d.ts

```diff
@@ -367,28 +376,37 @@ export interface InterceptorType {
     afterPoolConnection?: (
         connectionContext: ConnectionContextType,
         connection: DatabasePoolConnectionType
-    ) => MaybePromiseType<void>;
+    ) => MaybePromiseType<null>;
     afterQueryExecution?: (
         queryContext: QueryContextType,
         query: QueryType,
         result: QueryResultType<QueryResultRowType>
-    ) => MaybePromiseType<QueryResultType<QueryResultRowType>>;
+    ) => MaybePromiseType<null>;
     beforePoolConnection?: (
         connectionContext: PoolContextType
     ) => MaybePromiseType<DatabasePoolType | null | undefined>;
     beforePoolConnectionRelease?: (
         connectionContext: ConnectionContextType,
         connection: DatabasePoolConnectionType
-    ) => MaybePromiseType<void>;
+    ) => MaybePromiseType<null>;
     beforeQueryExecution?: (
         queryContext: QueryContextType,
         query: QueryType
-    ) => MaybePromiseType<QueryResultType<QueryResultRowType> | undefined>;
+    ) => MaybePromiseType<QueryResultType<QueryResultRowType> | null>;
+    beforeQueryResult?: (
+        queryContext: QueryContextType,
+        query: QueryType,
+        result: QueryResultType<QueryResultRowType>
+      ) => MaybePromiseType<null>;
+    beforeTransformQuery?: (
+        queryContext: QueryContextType,
+        query: QueryType
+    ) => MaybePromiseType<null>;
     queryExecutionError?: (
         queryContext: QueryContextType,
         query: QueryType,
         error: SlonikError
-    ) => MaybePromiseType<void>;
+    ) => MaybePromiseType<null>;
```

Comment:
Is this a change from v21? Could you link to the docs?

Reply:
https://github.com/gajus/slonik/blob/1ec5326171fa79135db93f92d35c5f8c628eb7fd/src/types.js#L468-L472

That is correct. It was one of the breaking changes.